streamalert_cli/manage_lambda/package.py

```diff
@@ -274,6 +274,7 @@ class AthenaPackage(LambdaPackage):
         'streamalert/shared'
     }
     package_name = 'athena_partition_refresh'
+    package_libs = {'netaddr'}
```

Comment:
Do you really need `netaddr`? I dont see it used in the partition refresh function